components/scripts/lib/build-scan-offline.sh

```diff
@@ -1,14 +1,10 @@
 #!/usr/bin/env bash
 
-readonly GRADLE_ENTERPRISE_LICENSE="${SCRIPT_DIR}/gradle-enterprise.license"
-readonly READ_BUILD_SCAN_DATA_JAR="${LIB_DIR}/build-scan-clients/read-build-scan-data-cmdline-tool-${SCRIPT_VERSION}-all.jar"
+readonly READ_BUILD_SCAN_DATA_JAR="${LIB_DIR}/build-scan-clients/build-scan-summary-${SUMMARY_VERSION}.jar"
 
 build_scan_dumps=()
 
 verify_offline_mode_required_files_exist() {
-  if [ ! -f "$GRADLE_ENTERPRISE_LICENSE" ]; then
-    die "ERROR: Missing required file gradle-enterprise.license in the root folder of the build validation scripts" "${INVALID_INPUT}"
-  fi
   if [ ! -f "$READ_BUILD_SCAN_DATA_JAR" ]; then
     die "ERROR: Missing required file to read the build scan data" "${INVALID_INPUT}"
   fi
```

Comment:
The above check can go away since the jar file will always be bundled in the BVS zip.

Reply:
Done in https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/487/commits/b9cb2f603e6c14a588d1b864c12ee0459872037f